### ISSUE TITLE
[Provider Request]: Intelichart based providers

### ISSUE THREAD

**Issue by madhacker3kxl** [fastenhealth/fasten-onprem#412]
### Provider/Institution Name

Intelichart

### Provider/Institution Website

https://www.intelichart.com

### Patient Portal Login Website

https://patientportal.intelichart.com/login/Account/Login

### Provider EHR Platform

Intelichart

### Anything else we should know?

Has a api webpage: https://openapi.intelichart.com/Help

**Comment by AnalogJ** [comment]
related #363 
related #298 

I've been trying to get ahold of intelichart since Sept 2023. 
- [openapihelp@intelichart.com](mailto:openapihelp@intelichart.com) results in a Mail delivery failure
- info@intelichart.com - no response
- hello@intelichart.com
- phone call resulted in a discussion and a message to email hello@intelichart.com

I just submitted an "Demo request" form on their partner page - https://www.intelichart.com/contact-us/

**Comment by madhacker3kxl** [comment]
Any update from intelichart?

**Comment by AnalogJ** [comment]
Not really. Emailed their CEO 3 days ago. I'll send a letter drafted by my lawyer next, and if that fails send a Information Blocking complaint to the ONC.

**Comment by AnalogJ** [comment]
Filed an Information Blocking request:

[IB-2819](https://inquiry.healthit.gov/support/plugins/servlet/desk/portal/6/IB-2819)

**Comment by AnalogJ** [comment]
related https://github.com/fastenhealth/fasten-sources/issues/81
